x-pack/plugins/security_solution/public/common/hooks/use_get_fields_data.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
This is currently only used to grab two fields in `x-pack/plugins/security_solution/public/detections/pages/alert_details/index.tsx`. The point of this file is to attempt to create a way of grabbing fields directly from the `fields` api. It's definitely a WIP, but happy to move this to a separate PR

Reply:
> This is currently only used to grab two fields in x-pack/plugins/security_solution/public/detections/pages/alert_details/index.tsx. The point of this file is to attempt to create a way of grabbing fields directly from the fields api. It's definitely a WIP, but happy to move this to a separate PR

Def your call. I wouldn't hold up this PR given the above, but it's hard to get a feel for it with the current level of test coverage.